curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/exercise-tracker.md

```diff
@@ -135,7 +141,47 @@ tests:
       }
     }
     "
-  - text: 'I can retrieve part of the log of any user by also passing along optional parameters of from & to or limit. (Date format yyyy-mm-dd, limit = int)'
+  - text: 'A request to `/api/exercise/log` includes a `count` key that is the number of exercises returned.'
```

Comment:
The test is actually making a request to the following in the `testString`:

const logRes = await fetch(url + `/api/exercise/log?userId=${_id}`);

so I think it should be made clear that the log is for specified user referenced by a queryString parameter userid.

```suggestion
  - text: 'A request to a user's log (`/api/exercise/log) returns an object with `count` property representing the number of exercises for the user.'
```